pool_lib/__init__.py

```diff
@@ -1019,22 +1017,13 @@ def _build_package_source(
                 build_outputdir,
             ]
             print(f"# {' '.join(command)}")
-            error = subprocess.run(command).returncode
+            deckdebuild_exitcode = subprocess.run(command).returncode
         verseek.seek_version(source_path)
 
-        if error:
+        if deckdebuild_exitcode != 0:
             msg = f"package `{package}' failed to build"
             if self.preserve_buildroot == "never":
                 shutil.rmtree(build_outputdir)
```

Comment:
I think this still needs to be removed

Reply:
(the ``preserve_buildroot == "never"`` check I mean)

Reply:
I've had a closer look at this one and while I don't think the behavior was quite right as it was, it was the same as before your previous changes. I have made a tweak though.